### DISCUSSION TITLE
TRITON_ENABLE_METRICS_CPU not found

### DISCUSSION THREAD

**Discussion by xc724655471** [triton-inference-server/server#5425] [Q&A] [answered]
**Description**
Is the flag TRITON_ENABLE_METRICS_CPU still in use, use docker to compile triton-server, no code about this flag can be found in the source code

**Triton Information**
What version of Triton are you using?
22.07

Are you using the Triton container or did you build it yourself?
build by source code

**To Reproduce**
Steps to reproduce the behavior.

cmake "-DTRT_VERSION=${TRT_VERSION}" "-DCMAKE_TOOLCHAIN_FILE=${CMAKE_TOOLCHAIN_FILE}" "-DVCPKG_TARGET_TRIPLET=${VCPKG_TARGET_TRIPLET}" "-DTRITON_TENSORFLOW_VERSION=2" "-DTRITON_TENSORFLOW_DOCKER_IMAGE=nvcr.io/nvidia/tensorflow:22.07-tf2-py3" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX:PATH=/tmp/tritonbuild/tensorflow2/install" "-DTRITON_COMMON_REPO_TAG:STRING=r22.07" "-DTRITON_CORE_REPO_TAG:STRING=r22.07" "-DTRITON_BACKEND_REPO_TAG:STRING=r22.07" "-DTRITON_ENABLE_GPU:BOOL=ON" "-DTRITON_ENABLE_MALI_GPU:BOOL=OFF" "-DTRITON_ENABLE_STATS:BOOL=ON" "-DTRITON_ENABLE_METRICS:BOOL=ON" ..

**Comment by the-david-oy** [comment, marked as answer]
Yes. You can see it used in several places in the core repo for enabling the collection of CPU metrics. One good place to look is [here](https://github.com/triton-inference-server/core/blob/main/src/metrics.cc#L153-L166).

**Reply by xc724655471** [reply]
Thanks for reply, I solved this problem by upgrading the Triton Server version. CPU Metric is the new feature of Release 2.26, with a container 22.09.
